src/NServiceBus.Log4Net/Log4NetFactory.cs

```diff
@@ -6,6 +6,10 @@
     /// <summary>
     /// Configure NServiceBus logging messages to use Log4Net. Use by calling <see cref="LogManager.Use{T}"/> the T is <see cref="Log4NetFactory"/>.
     /// </summary>
+    [ObsoleteEx(
+        Message = "Support for external logging providers is no longer provided by NServiceBus providers for each logging framework. Instead, the NServiceBus.Extensions.Logging library provides the ability to use any logging provider that conforms to the Microsoft.Extensions.Logging abstraction.",
```

Comment:
Similar to the other 2 PRs, I was going to suggest a change to this message as well, but I can't find an actual official integration package for Log4Net.

The only one I can find is https://www.nuget.org/packages/Microsoft.Extensions.Logging.Log4Net.AspNetCore/. Are we expecting people to use that even though it's just a random package and not from log4net?

If so, here's my suggestion for the change:

```suggestion
        Message = "NServiceBus is now providing support for logging libraries through the Microsoft.Extensions.Logging abstraction. Remove the NServiceBus.NLog package. Install the NServiceBus.Extensions.Logging and Microsoft.Extensions.Logging.Log4Net.AspNetCore packages instead.",
```